### PR TITLE
fix SYNOPSIS alphabetical order man-create

### PR DESCRIPTION
**Pull request by sallyom**
Signed-off-by: Sally O'Malley somalley@redhat.com

### CODE REVIEW

**Comment by sallyom** [comment]
@rhatdan

File: man/docker-create.1.md [hunk 1]
```diff
@@ -107,18 +107,18 @@ The initial status of the container created with **docker create** is 'created'.
 then processes in your Docker container will only use memory from the first
 two memory nodes.
 
-**-cpu-quota**=0
+**--cpu-quota**=0
```
Comment (thaJeztah) [inline man/docker-create.1.md]: good catch!

**Comment by thaJeztah** [comment]
Great! You made my OCD side happy :smile:

LGTM

ping @fredlf @jamtur01 @moxiegirl @SvenDowideit

**Comment by moxiegirl** [comment]
LGTM Thank you @sallyom nice!

**Pull request merged**